Cargo.lock

```diff
@@ -507,7 +555,7 @@ dependencies = [
 [[package]]
 name = "miniconf_mqtt"
 version = "0.9.0"
-source = "git+https://github.com/quartiq/miniconf#af999890beddf1fc920ae6706a3861288000cab9"
+source = "git+https://github.com/quartiq/miniconf#15c2bb99abacdad2ea00236b9deae08b6ac91a23"
```

Comment:
Do you think we're at a point where we can release miniconf, or should we test it out a bit here first?

Reply:
We should release but I'd like to run it through here and FLS first.

Reply:
Worked nicely with attenuation validation.